integration_testing/features/admin/admin-change-facility-settings.feature

```diff
@@ -1,10 +1,10 @@
 Feature: Admin change facility settings
-    Admin needs to be able to change the user sign-in/up, self-edit, and content download options according to the needs of the facility 
+  Admin needs to be able to change the user sign-in/up, self-edit, and content download options according to the needs of the facility
```

Comment:
were all the formatting changes to feature files supposed to be part of this PR?

Reply:
whoops I see your note above